### DISCUSSION TITLE
Raise plugin window without rendering

### DISCUSSION THREAD

**Discussion by manatactical** [apohl79/audiogridder#632] [Ideas]
Hi,

For people running many screens, it would be great to have a check option to only raise the plugin window on the server machine screen WITHOUT rendering it on the plugin side machine. 
I have one mouse for both of my computers (mac and pc) and it could save some bandwidth doing so. Having both screens with the same plugin GUI is redundant in my case.

Vienna Ensemble Pro has this kind of feature for the instances. Very useful.

What do you guys think?

Keep up the good work.

**Comment by CaBuffalo** [comment]
Yes, you can do it by disabling the screen capture on the server.

On Fri, Jul 2, 2021 at 12:10 PM Manabruuury ***@***.***>
wrote:

> Hi,
>
> For people running many screens, it would be great to have a check option
> to only raise the plugin window on the server machine screen WITHOUT
> rendering it on the plugin side machine.
> I have one mouse for both of my computers (mac and pc) and it could save
> some bandwidth doing so. Having both screens with the same plugin GUI is
> redundant in my case.
>
> Vienna Ensemble Pro has this kind of feature for the instances. Very
> useful.
>
> What do you guys think?
>
> Keep up the good work.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/632>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CS4GIB5HKQV6NEGTTTVYFKBANCNFSM47XHRVFQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by manatactical** [reply]
Oh my god, didn't know that! Thanks :D

**Reply by Oemfoefoe-1984** [reply]
hi m8

how do you use one mouse with 2 machines? Im using macbook pro and win10 as server...but i have to use 2 mouses lol

**Reply by manatactical** [reply]
I'm using Synergy. It's only $29 bucks, and it's a life saver!

**Reply by rob-88-brigham** [reply]
I use barrier. It's free, it's the open source fork of synergy. I asked same thing about not opening that window. I also use multiple computers. Whole lot cheaper to use a bunch of 2nd hand computers than one new one. My main computer is a 2007 Mac pro, maxed out running Yosemite. I have no intention of upgrading. 😃 Gridder has significantly improved my work flow.

**Reply by rob-88-brigham** [reply]
okay, but I still have a little window that I cannot close or hide without closing it on the computer the plug is running on.  It's a little better.
